### PR TITLE
Some devil fixes for bugs I noticed when I forced the gamemode and ruined a round

### PR DESCRIPTION
**Pull request by TheRyeGuyWhoWillNowDie**
Fixes some bugs
Makes it substantially more obvious that you are not an antagonist after selling your soul

# Changelog

:cl:  
tweak: Some devil features have received QoL
bugfix: Some devil bugs have been fixed
/:cl:

### CODE REVIEW

**Review by Simplehorror — APPROVED** [review]

File: code/modules/antagonists/devil/devil.dm [hunk 1]
```diff
@@ -459,6 +464,7 @@ GLOBAL_LIST_INIT(devil_suffix, list(" the Red", " the Soulless", " the Master",
 		H.equip_to_slot_or_del(new /obj/item/clothing/shoes/laceup(H), SLOT_SHOES)
 		H.equip_to_slot_or_del(new /obj/item/storage/briefcase(H), SLOT_HANDS)
 		H.equip_to_slot_or_del(new /obj/item/pen(H), SLOT_L_STORE)
+		H.equip_to_slot_or_del(new /obj/item/clothing/glasses/sunglasses(H), SLOT_GLASSES)
```
Comment (ToasterBiome) [inline code/modules/antagonists/devil/devil.dm]: I'm sussing among us

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/devil/devil.dm (hunk 1 above)
Comment (TheRyeGuyWhoWillNowDie) [reply code/modules/antagonists/devil/devil.dm]: QoL

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
no sunglasses

**Comment by TheRyeGuyWhoWillNowDie** [comment]
ok

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**